### ISSUE TITLE
Crashes when searching for DNS lookup

### ISSUE THREAD

**Issue by jonasdev** [madsobel/fawkes#10]
Searching for jonas-k.me = nothing happens

Searching for google.com =
![image](https://user-images.githubusercontent.com/26645731/142375842-b1a133a1-bb01-4d87-9945-0af84b222af9.png)

**Comment by madsobel** [comment]
Hi @jonasdev 

Auch! I think I know what's going on. Let me check it, and get back to you.

**Comment by madsobel** [comment]
Hi again @jonasdev 

So this is "fixed" now.
There were two things going on here.
1) I recently added so that if the only thing that is being rendered in a table cell is a link, I wrapped it in an anchor tag. However I did not account for the fact that non-string values could/would be added as values, which is the case with a DNS records TTL.

2) It seems like it still dosen't work for your domain "jonas-k.me". I've added some more servers to ask, but still seem like it dosen't find any. I will have to investigate this further. So for now, I've added a text explaining that nothing was found, so you at least not get the impression that nothing happens.

Thanks for using Fawkes!
